lib/validator/validator.go

```diff
@@ -230,10 +289,7 @@ func validateScopeGlobalOrNode(v *validator.Validate, topStruct reflect.Value, c
 func validatePolicyType(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
 	s := field.String()
 	log.Debugf("Validate policy type: %s", s)
-	if s == string(api.PolicyTypeIngress) {
-		return true
-	}
-	if s == string(api.PolicyTypeEgress) {
+	if s == string(api.PolicyTypeIngress) || s == string(api.PolicyTypeEgress) {
```

Comment:
Should this be the uppercased values of `Ingress` and `Egress` (as it is done here and according to k8s docs https://kubernetes.io/docs/concepts/services-networking/network-policies/) or lowercased (as per the docs https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy)?

@bcreane @robbrockbank @caseydavenport  WDYT?

Reply:
yeah, we'll go with `Ingress` and `Egress` just like k8s.

Reply:
done